bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
Hmm.. Doesn't [this line](https://github.com/amplab/tachyon/pull/2501/files#diff-40c38fb954cae1e4d8ca66ae6f3d7a52L17) set `bin`?